tests/ui/perfherder/alerts_test.jsx

```diff
@@ -202,6 +203,43 @@ test('toggle buttons should filter alert summary and alerts by selected filter',
   expect(alert2).not.toBeInTheDocument();
 });
 
+test('alert filtering ignores repository and options, but filters by platform', async () => {
+  const { getByPlaceholderText, getByTestId } = alertsView();
+  const alertsFilterInput = getByPlaceholderText(filterText.inputPlaceholder);
+
+  const alertSummary1 = await waitForElement(() =>
+    getByTestId(`alert summary ${testAlertSummaries[0].id.toString()} title`),
+  );
+  const alertSummary2 = await waitForElement(() =>
+    getByTestId(`alert summary ${testAlertSummaries[1].id.toString()} title`),
+  );
+
+  // alertSummary2's alerts
+  const alert1 = await waitForElement(() => getByTestId('69526'));
+  const alert2 = await waitForElement(() => getByTestId('69530'));
+
+  expect(alertsFilterInput).toHaveClass('form-control');
+  expect(alertSummary1).toBeInTheDocument();
+  expect(alertSummary2).toBeInTheDocument();
+  expect(alert1).toBeInTheDocument();
+  expect(alert2).toBeInTheDocument();
+
+  fireEvent.change(alertsFilterInput, {
+    target: {
+      value: `
+      ${repos[0].name} 
+      ${testAlertSummaries[1].alerts[0].series_signature.extra_options[0]} 
```

Comment:
We should also specify the `options` here, alongside the repository name.

Reply:
Done.

Reply:
`options` are specific to the series signature, not to repositories.

Reply:
OK, I will change and access the `options` like this:
```
testAlertSummaries[0].alerts[0].series_signature.options[0]
```